src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int ran = new Random().nextInt(5);
+        int a = new Random().nextInt(10) + 2;
+        int b = new Random().nextInt(10) + 2;
+        int c = new Random().nextInt(10) + 2;
+        switch (ran) {
+            case (0):

```

Comment:
```suggestion
            case 0:
```
the same for other cases